examples/pending-order-reminder/Ballerina.toml

```diff
@@ -6,3 +6,9 @@ distribution = "2201.9.0"
 
 [build-options]
 observabilityIncluded = true
+
+[[dependency]]
+org = "ballerinax"
+name = "sap"
+version = "1.0.0"
```

Comment:
isn't it 1.0.1 ?

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerinax-sap/pull/20